frontend/public/components/pdb/PodDisruptionBudgetField.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { useTranslation } from 'react-i18next';
+import {
+  PodDisruptionBudgetKind,
+  K8sResourceKind,
+  referenceForModel,
+} from '@console/internal/module/k8s';
+import { useK8sWatchResource } from '../utils/k8s-watch-hook';
+import { ResourceLink, DetailsItem, getPDBResource } from '../utils';
+import { PodDisruptionBudgetModel } from '../../models';
+
+const AvaiallabilityRequirement: React.FC<AvaiallabilityRequirementProps> = ({ pdb, replicas }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      {typeof pdb?.spec?.minAvailable !== 'undefined'
+        ? t('public~Min Available {{minAvailable}} of {{count}} pod', {
+            minAvailable: pdb.spec.minAvailable,
+            count: replicas,
+          })
+        : t('public~Max Unavailable {{maxUnavailable}} of {{count}} pod', {
+            maxUnavailable: pdb?.spec?.maxUnavailable ?? 1,
+            count: replicas,
+          })}
+    </>
+  );
+};
+
+export const PodDisruptionBudgetField: React.FC<PodDisruptionBudgetFieldProps> = ({ obj }) => {
+  const { t } = useTranslation();
+  const [pdbResources] = useK8sWatchResource<PodDisruptionBudgetKind[]>({
+    kind: referenceForModel(PodDisruptionBudgetModel),
+    isList: true,
+    namespaced: true,
+    namespace: obj.metadata.namespace,
+  });
+  const pdb = getPDBResource(pdbResources, obj);
+
+  return (
+    <DetailsItem label={t('public~PodDisruptionBudgets')} obj={obj} path="">
```

Comment:
Is path optional? It would be better to leave it off rather than setting to `""`

```suggestion
    <DetailsItem label={t('public~PodDisruptionBudgets')} obj={obj}>
```